packages/lib/src/vis/models.ts

```diff
@@ -90,3 +90,9 @@ export interface HistogramParams {
   colorMap?: ColorMap;
   invertColorMap?: boolean;
 }
+
+export enum ModifierKey {
+  Alt = 'Alt',
+  Ctrl = 'Control',
+  Shift = 'Shift',
+}
```

Comment:
Modifier keys can be any string but I thought it was better to restrict them at first

Reply:
Might be simpler to use a union `'Alt' | 'Control' | 'Shift'`. Avoids having to import the enum in the consumer component.